languagetool-language-modules/fr/src/main/resources/org/languagetool/rules/fr/grammar.xml

```diff
@@ -96630,16 +96639,33 @@ Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA 02110-1301, USA
                     <token regexp="yes" skip="5">que|qu'</token>
                     <token inflected="yes" min="0" max="1">ne</token>
                     <token postag="R pers obj.*" postag_regexp="yes" min="0" max="3"/>
+                    <token postag="R pers suj 3 e s" min="0" max="1"/>
                     <token postag="V avoir.*" postag_regexp="yes" skip="4"/>
                     <token postag="V.* ppa.* s?p|V.* ppa.* m sp?" postag_regexp="yes"/>
                     <token postag="D.*" postag_regexp="yes"/>
+                    <example>Ce n'est pas en améliorant la bougie que l'on a inventé l'ampoule.</example>
+                </antipattern>
+                <antipattern>
+                    <token postag="D f s"/>
+                    <token postag="J f s|C coor|A" postag_regexp="yes" min="0" max="4"/>
+                    <token postag="N f s"/>
+                    <token postag="J f s|C coor|A" postag_regexp="yes" min="0" max="4"/>
+                    <token postag="[DP].*" postag_regexp="yes"/>
+                    <token postag="J m p|N m p" postag_regexp="yes" min="1" max="4"/>
+                    <token regexp="yes" skip="5">que|qu'</token>
+                    <token inflected="yes" min="0" max="1">ne</token>
+                    <token postag="R pers obj.*" postag_regexp="yes" min="0" max="3"/>
+                    <token postag="R pers suj 3 e s" min="0" max="1"/>
+                    <token postag="V avoir.*" postag_regexp="yes" skip="4"/>
+                    <token postag="V.* ppa.* s?p|V.* ppa.* m sp?" postag_regexp="yes"/>
+                    <example>Je ne récolte que la sève des arbres que j'ai moi-même plantés.</example>
                 </antipattern>
                 <pattern>
                     <token negate="yes" regexp="yes">plus|moins|avec|et|dans|de|d'|au|à</token>
                     <token postag="D f s"/>
                     <token postag="J f s|C coor|A" postag_regexp="yes" min="0" max="4"/>
                     <token postag="N f s">
-                        <exception regexp="yes">semaine|manière|année|raison</exception>
+                        <exception regexp="yes">collection|semaine|manière|année|raison|série|ensemble|gamme|assortiment|lot|palette|variété|choix|répertoire|éventail</exception>
```

Comment:
Still QUE_AVOIR[5], I found a FP with "collection" so I went for more words related to sets